kms/mackms/mackms_test.go

```diff
@@ -0,0 +1,705 @@
+//go:build darwin && cgo && !nomackms
```

Comment:
This probably needs the CI to be adapted to also run on macOS

Reply:
This is something I considered doing but I decided to leave it for a future PR.